addons/event/models/event_event.py

```diff
@@ -236,7 +237,9 @@ def _default_question_ids(self):
         selection=[
             ('A4_french_fold', 'A4 foldable'),
             ('A6', 'A6'),
-            ('four_per_sheet', '4 per sheet')
+            ('four_per_sheet', '4 per sheet'),
+            ('96x82', '96x82mm (Badge Printer)'),
+            ('96x134', '96x134mm (Badge Printer)')
```

Comment:
Adding new options to a selection field requires an update / to create the `ir_model_fields_selection` manually.
e.g.
```sql
INSERT INTO ir_model_fields_selection(field_id, value, name) VALUES((SELECT id FROM ir_model_fields WHERE name = 'badge_format'), '96x82', '{"en_US": "96x82mm (Badge Printer)"}');
INSERT INTO ir_model_fields_selection(field_id, value, name) VALUES((SELECT id FROM ir_model_fields WHERE name = 'badge_format'), '96x134', '{"en_US": "96x134mm (Badge Printer)"}');
```

It is acceptable but I am still writing it down to not forget.